src/utils/navState.js

```diff
@@ -0,0 +1,15 @@
+export const isNavClosed = () => {
+  if (window.localStorage) {
```

Comment:
might be worth it to check if `typeof window !== undefined` here before accessing localStorage